src/components/TaskForm/CreateTaskDialog.tsx

```diff
@@ -47,11 +48,27 @@ export function CreateTaskDialog({ projectId }: Props) {
         return
       }
 
-      setOpen(false)
       toast.success("Task created")
+
+      // If Auto-Create is not checked, close the dialog
+      if (!autoCreate) {
+        setOpen(false)
+      }
     })
   }
 
+  const Checkbox = () => (
+    <div className="flex items-center gap-2">
+      <input
+        type="checkbox"
+        id="autoCreate"
+        checked={autoCreate}
+        onChange={(e) => setAutoCreate(e.target.checked)}
+      />
+      <label htmlFor="autoCreate">Auto-Create</label>
+    </div>
+  )
```

Comment:
There's already a Checkbox component in `src/components/ui`, let's use that one :)

Reply:
Sure! I'll replace the current checkbox with the one from src/components/ui for consistency. I'll also handle the form reset after submission when the checkbox is enabled and include it in this PR.

Reply:
Have you tried to logout and login again on the browser?

I don't think it's an issue in your code anyway

Reply:
> Have you tried to logout and login again on the browser?
> 
> I don't think it's an issue in your code anyway

Yes, that worked. Thank you